api/src/main.ts

```diff
@@ -16,10 +16,10 @@ setupMiddleware(app)
 
 const router = Router()
 router.get("/", (_, res) => res.send({ status: "ok" }))
-router.get("/sso/metabase", restrict, metabaseAuthHandler)
-router.get("/products", restrict, productListHandler)
-router.get("/categories", restrict, categoryListHandler)
-router.get("/product/:id", productDetailHandler)
+router.get("/sso/metabase", metabaseAuthHandler)
+router.get("/products", withCacheHeader, productListHandler)
+router.get("/categories", withCacheHeader, categoryListHandler)
+router.get("/product/:id", withCacheHeader, productDetailHandler)
```

Comment:
Use the `withCacheHeader` middleware on cacheable data endpoints. These endpoints are all safe to cache now, as we got rid of cookies.